src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve();
+    }
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve();
+    }
+  });
+});
+
+const thirdPromise = new Promise((resolve, reject) => {
+  let leftClick = false;
+  let rightClick = false;
+
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      leftClick = true;
+    }
+
+    if (e.button === 2) {
+      rightClick = true;
+    }
+
+    if (leftClick && rightClick) {
+      resolve();
+    }
+  });
+});
+
+firstPromise
+  .then(() => {
+    const newDiv = document.createElement('div');
+
+    newDiv.setAttribute('data-qa', 'notification');
+    newDiv.setAttribute('class', 'success');
+    newDiv.textContent = 'Firs promise was resolved';
```

Comment:
There is a typo in the text content of the notification. It should be 'First promise was resolved' instead of 'Firs promise was resolved'.